packages/@ionic/lab/.gitignore

```diff
@@ -1 +1,2 @@
 www
+bin/ionic-lab.*
```

Comment:
That's strange, which file(s) are generated?

Reply:
generate `ionic-lab.3077068012`. The contents are the same as `ionic-lab`.

Reply:
Did it do this with any of the other packages with binaries? `v1-toolkit`, `cli-scripts`? Just curious.

Reply:
I retry new project same operation, but don't make this file.
Probably, If node-sass error occurs, this file made.